tests/Doctrine/Tests/DBAL/Schema/ColumnTest.php

```diff
@@ -56,6 +57,17 @@ public function testToArray()
     }
 
     /**
+     * @group legacy
```

Comment:
I was not able to use the `simple-phpunit` binary, which means I can test the deprecation message as described here: https://symfony.com/doc/current/components/phpunit_bridge.html#write-assertions-about-deprecations